nixos/modules/services/x11/display-managers/default.nix

```diff
@@ -126,6 +126,14 @@ let
         (*) echo "$0: Desktop manager '$desktopManager' not found.";;
       esac
 
+      ${optionalString cfg.startDbusSession ''
```

Comment:
@cruegge Instead of doing this in the default xsession, could we only do it for the desktop managers that require it? For example, KDE 5 uses `dbus-update-activation-environment` where available, or it falls back onto its own `ksyncdbusenv` if not.